src/irmin-pack/irmin_pack.mli

```diff
@@ -24,6 +24,7 @@ val config :
 
 module Dict = Dict
 module Pack = Pack
+module Index = Pack_index
```

Comment:
This is being exposed here solely to test the `Pack` interface (since the constructor now needs an `Index`). Maybe it should be moved to a `Private` namespace or similar?